docs/introduction.mdx

```diff
@@ -61,12 +61,12 @@ The ["@emotion/core"](https://www.npmjs.com/package/@emotion/core) package requi
 
 - Best when using React with a build environment that can be configured.
 
-- CSS prop support
-
-  - Similar to the `style` prop but adds support for nested selectors, media queries, and auto-prefixing.
+- `css` prop support
 
   - Allows developers to skip the `styled` API abstraction and style components and elements directly.
 
+  - Similar to the `style` prop, but also has support for auto vendor-prefixing, nested selectors, and media queries.
```

Comment:
Would prefer moving this to its former position (first in the list) - seems more important than the point about styled API.